readme.md

```diff
@@ -315,7 +315,20 @@ In addition to `redux` --> `mobx-state-tree`, we've also transitioned to using `
 
 In Bowser, TypeScript is fully set up, so if you know TS, all you need to do is start coding!
 
-### Resources
+## GraphQL comparing to RESTful
+
+### Design/Architecture
+RESTful API uses HTTP verbs (Post, Get, Patch, etc...) in order to manipulate data. 
+Because of this, designing a RESTful system requires creating endpoints for each request such as: POST/user, GET/user, DELETE/user etc... 
```

Comment:
@JonnoKiwi. This sentence doesn't seem correct. Verbs are an inherit property of HTTP. Your sentence makes it seem like you have to create separate endpoints for each Verb

Reply:
I've updated my explanation